metaci/release/models.py

```diff
@@ -46,11 +47,22 @@ class Release(StatusModel):
     )
 
     release_creation_date = models.DateField(
-        _("release creation date"), null=True, blank=True
+        _("release creation date"),
+        null=True,
+        blank=True,
+        default=datetime.date.today().isoformat(),
+    )
+    sandbox_push_date = models.DateField(
+        _("sandbox push date"),
+        null=True,
+        blank=True,
+        default=(datetime.date.today() + datetime.timedelta(days=6)).isoformat(),
     )
-    sandbox_push_date = models.DateField(_("sandbox push date"), null=True, blank=True)
     production_push_date = models.DateField(
-        _("production push date"), null=True, blank=True
+        _("production push date"),
+        null=True,
+        blank=True,
+        default=(datetime.date.today() + datetime.timedelta(days=6)).isoformat(),
```

Comment:
These defaults are not working right because they are calculated when the module is imported, rather than when a Release record is added. You need to set `default` to a function which returns the date, instead of setting a date value directly.

Also, sandbox date should default to the day the release is created instead of the same day as production push, right?

Reply:
@davisagli ah good catch then.